static/sw.js

```diff
@@ -90,8 +90,8 @@ function requestAccepts(request, contentType) {
 function ampByExampleHandler(request, values) {
   // for samples show offline page if offline and samples are not cached
   if (requestAccepts(request, 'text/html')) {
-    // never use cached version for AMP CORS requests (e.g. amp-live-list)
-    if (request.url.indexOf("__amp_source_origin") != -1) {
+    // never use cached version for AMP CORS requests (e.g. amp-live-list) or pages open from href like shopping_cart
+    if (request.url.indexOf("__amp_source_origin") != -1 || request.url.indexOf("shopping_cart") != -1) {
```

Comment:
Are URL variable substitutions already supported? In this case you could simply use the RANDOM variable substitution in the URL, e.g.:

`href="<%host%>/shopping_cart/?clientId={{ClientId}}&"`

Otherwise: can you make this more generic so that we can use it in other samples as well, e.g. add a query parameter `shopping_cart/?cache=0`.